app/helpers/view_components/mentor/header.rb

```diff
@@ -47,43 +47,60 @@ def stats
         end
       end
 
-      def tabs
-        [
-          link_to(
-            Exercism::Routes.mentoring_inbox_path,
-            class: tab_class(:workspace)
-          ) do
-            graphical_icon(:overview) +
-              tag.span("Your Workspace") +
-              tag.span(number_with_delimiter(inbox_size), class: 'count')
-          end,
-
-          link_to(
-            Exercism::Routes.mentoring_queue_path,
-            class: tab_class(:queue)
-          ) do
-            graphical_icon(:queue) +
-              tag.span("Queue") +
-              tag.span(number_with_delimiter(queue_size), class: 'count')
-          end,
-
-          link_to(
-            Exercism::Routes.mentoring_testimonials_path,
-            class: tab_class(:testimonials)
-          ) do
-            graphical_icon(:testimonials) +
-              tag.span("Testimonials") +
-              tag.span(number_with_delimiter(num_testimonials), class: 'count')
-          end,
+      def tabs = [workspace_tab, queue_tab, testimonials_tab, automation_tab]
+
+      def workspace_tab
+        link_to(
+          Exercism::Routes.mentoring_inbox_path,
+          class: tab_class(:workspace)
+        ) do
+          graphical_icon(:overview) +
+            tag.span("Your Workspace") +
+            tag.span(number_with_delimiter(inbox_size), class: 'count')
+        end
+      end
 
-          tag.div(
+      def queue_tab
+        link_to(
+          Exercism::Routes.mentoring_queue_path,
+          class: tab_class(:queue)
+        ) do
+          graphical_icon(:queue) +
+            tag.span("Queue") +
+            tag.span(number_with_delimiter(queue_size), class: 'count')
+        end
+      end
+
+      def testimonials_tab
+        link_to(
+          Exercism::Routes.mentoring_testimonials_path,
+          class: tab_class(:testimonials)
+        ) do
+          graphical_icon(:testimonials) +
+            tag.span("Testimonials") +
+            tag.span(number_with_delimiter(num_testimonials), class: 'count')
+        end
+      end
+
+      def automation_tab
+        unless current_user.supermentor? || current_user.admin?
+          return tag.div(
             class: "#{tab_class(:automation)} locked",
             'aria-label': "This tab is locked"
           ) do
             graphical_icon(:automation) +
-              tag.span("Automation")
+            tag.span("Automation")
           end
-        ]
+        end
```

Comment:
This should have a tooltip (as per figma) that explains why it's locked and how to unlock it. It's not blocking, but I'd like to ensure we eventually get that (if we don't have it already!)

Reply:
I've created a tooltip, but the figma version is outdated as it uses reputation as the unlock mechanism, whereas we're actually using num_finished_mentoring_sessions >= 100 and satisfaction_percentage >= 95.